config/prow/cluster/boskos_janitor.yaml

```diff
@@ -0,0 +1,40 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: boskos-janitor-nongke
```

Comment:
Was just wondering if the word `nongke` had any significance here :)
Shall we just have the name `boskos-janitor` for our infra?
Also, @mkumatag do you think we need 12 replicas of this?

Reply:
I'm okay removing nongke from there.. we don't need 12 replicas.

Reply:
@Rajalakshmi-Girish, changed the number of replicas to 2.